app/models/payment.rb

```diff
@@ -189,7 +189,56 @@ def sync_with_payout_processor
     sync_with_paypal if processor == PayoutProcessorType::PAYPAL
   end
 
+  def as_json(options = {})
+    json = {
+      id: external_id,
+      amount: format("%.2f", (amount_cents || 0) / 100.0),
+      currency: currency,
+      status: state,
+      created_at: created_at,
+      processed_at: state == COMPLETED ? updated_at : nil,
+      payment_processor: processor,
+      bank_account_visual: bank_account&.account_number_visual,
+      paypal_email: payment_address
+    }
+
+    if options[:include_sales]
+      json[:sales] = successful_sales.map { |sale| sale.as_json(version: 2) }
+      json[:refunded_sales] = refunded_sales.map { |sale| sale.as_json(version: 2) }
+      json[:disputed_sales] = disputed_sales.map { |sale| sale.as_json(version: 2) }
```

Comment:
```suggestion
      json[:sales] = successful_sales.map(&:external_id)
      json[:refunded_sales] = refunded_sales.map(&:external_id)
      json[:disputed_sales] = disputed_sales.map(&:external_id)
```
Since a payout can include a lot of sales, let's just list the sale IDs here. We have a sales API which can be used to get further details of the included sales if needed.